tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -305,6 +305,15 @@
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
-    oc wait --for condition=Ready osdpns/openstack --timeout=40m
-  # TODO: work on network configuration for making possible to run this task on other IP ranges
-  when: "edpm_node_ip.startswith('192.168.122')"
+    oc get pod -l app=openstackansibleee
+    NODESET_NAME=openstack
+    LAST_SERVICE_NAME=$(oc get osdpns ${NODESET_NAME} -o jsonpath='{.spec.services}' | jq -r last)
+    echo ${LAST_SERVICE_NAME}
+    LAST_SERVICE_POD=$(oc get pod -l app=openstackansibleee -o name | grep "${LAST_SERVICE_NAME}-${NODESET_NAME}")
+    echo ${LAST_SERVICE_POD}
+    oc get ${LAST_SERVICE_POD} -o jsonpath='{.status.phase}{"\n"}' | grep Succeeded
+    oc get osdpd openstack -o jsonpath='{.status.deployed}{"\n"}' | grep true
+  register: osdpd_running_result
+  until: osdpd_running_result is success
+  retries: 40
+  delay: 60
```

Comment:
is that 3200sec of waiting? let's shorten it to not keep CI workers hurdling on waiting

Reply:
I'm just keeping the same 40 minutes as before (not sure how much it can be reduced, maybe 30 minutes)